src/main/java/xyz/nucleoid/extras/player_list/PlayerListHelper.java

```diff
@@ -18,14 +30,14 @@ public static GameMode getGameMode(ServerPlayerEntity player, boolean gray) {
 
     public static PlayerListS2CPacket getUpdatePacket(ServerPlayerEntity player, PlayerListS2CPacket.Action action, boolean gray) {
         var packet = new PlayerListS2CPacket(action);
-        packet.getEntries().add(new PlayerListS2CPacket.Entry(player.getGameProfile(), 0, getGameMode(player, gray), getDisplayName(player, gray)));
+        packet.getEntries().add(new PlayerListS2CPacket.Entry(player.getGameProfile(), 0, getGameMode(player, gray), getDisplayName(player, gray), getPublicKey(player)));
```

Comment:
Nit: the key technically won't be encoded here at all for an update packet, might be worth just passing null